Makefile

```diff
@@ -55,10 +55,10 @@ all: $(UPTODATE_FILES) binaries
 check: all lint unit-tests container-tests
 
 BINARIES = \
-	cmd/wksctl/wksctl \
-	cmd/mock-authz-server/server \
-	cmd/mock-https-authz-server/server \
-	cmd/controller/controller \
+	bin/wksctl \
+	bin/mock-authz-server \
+	bin/mock-https-authz-server \
+	bin/controller \
```

Comment:
Is there a big benefit from this change?  I think it makes things less consistent with other projects.

Reply:
I personally prefer bin for binaries, I think that's much easier to find than individual `cmd/*` locations, also more to type. A quick look yielded:

bin/
 - footloose
 - flagger
 - ignite

/ (top-level)
 - eksctl

cmd/*
 - scope
 - weave-net

So I'm not sure which one is the right way, but coming from the k8s world I'd expect things in `bin`. I quickly checked CAPI core, CAPA, CAPZ and CAPDO, and they all use  `bin/`.

Regardless of what we choose (to be decided), I'd like us to become more consistent in wrt these things.

Reply:
I'd be open to making it consistent with CAP* projects.
@richardcase had some other points here, e.g. using Kustomize, and about where the manifests are located.